modules/recommended-etcd-practices.adoc

```diff
@@ -25,7 +30,12 @@ The following hard disk features provide optimal etcd performance:
 * RAID 0 technology for increased performance.
 * Dedicated etcd drives. Do not place log files or other heavy workloads on etcd drives.
 
-Avoid NAS or SAN setups and spinning drives. Always benchmark by using utilities such as fio. Continuously monitor the cluster performance as it increases.
+[NOTE]
+====
+Avoid NAS or SAN setups and spinning drives. Ceph Rados Block Device (RBD), as well as other network-attached storage, can result in unpredictable network latency. To provide fast storage to etcd nodes at scale, use PCI passthrough to pass NVM devices directly to the nodes. 
```

Comment:
One more nit from my side:

as well as > and

![Screenshot from 2023-06-27 10-53-53](https://github.com/openshift/openshift-docs/assets/57954076/90a757d5-07e9-41e8-af47-d2d05aa35aee)

Reply:
Feel free to just use the GH suggestion feature here, but yeah, changing.

```suggestion
Avoid NAS or SAN setups and spinning drives. Ceph Rados Block Device (RBD) and other types of network-attached storage can result in unpredictable network latency. To provide fast storage to etcd nodes at scale, use PCI passthrough to pass NVM devices directly to the nodes. 
```